package.json

```diff
@@ -38,7 +38,8 @@
     "ember-export-application-global": "^1.0.5",
     "ember-load-initializers": "^0.5.1",
     "ember-resolver": "^2.0.3",
-    "loader.js": "^4.0.1"
+    "loader.js": "^4.0.1",
+    "phantomjs-prebuilt": "^2.1.14"
```

Comment:
I don't know if this is needed.  As per ember-cli, you can easily install this globally to get this to work.  https://ember-cli.com/user-guide/#phantomjs